### ISSUE TITLE
FreeDV & "DATA1 Function" in ICOM7100

### ISSUE THREAD

**Issue by Nokia808** [drowe67/freedv-gui#699]
Hello. 

In ICOM7100: setting > Connectors > USB2/DATA1 Function > DATA1 Function, 

In "DATA1 Function" there are 4 choices: OFF, RTTY Decode, DV Data, GPS -see attached screenshot.

![DATA1-Function](https://github.com/drowe67/freedv-gui/assets/25375330/5e1696fc-1639-413f-acf9-b2eb677ab917)

Currently, I set "DATA1 Function" at DV data. Is this really related to FreeDV or not ? Can I set it at "OFF" & using FreeDV ? As long as "DV Data baud" not related to FreeDV, so setting of "DATA1 at "DV Data" should be unrelated, isn't it ?

I asked this because I'm working on other program JS8Call that depend on FT8, & FT8 need DATA1 Function to be set at OFF. So, if "DATA1 Function" at "DV data" not needed for FreeDV, then I can set it at "OFF" permanently & no need to change it from time to time ....

**Comment by tmiw** [comment]
Yeah, DATA1 can be set to OFF for FreeDV as well. It only does CAT control and audio and doesn't use GPS or any of the other options available for DATA1. Let me know if you have any other questions!

**Comment by Nokia808** [comment]
Many thanks. Question is answered & I will close this issue.
